server/services/user.js

```diff
@@ -43,15 +43,18 @@ const userServiceFactory = (elite2Api, oauthApi) => {
       }
     }
 
+    const prisons = config.useOfForce.prisons.split(',')
```

Comment:
meh.. could call this `useOfForcePrisons`. `prisons` is a bit too generic.

Reply:
ok